core/domain/user_services_test.py

```diff
@@ -18,15 +18,38 @@
 import os
 
 from core.domain import collection_services
+from core.domain import event_services
 from core.domain import exp_services
 from core.domain import rights_manager
+from core.domain import user_jobs_continuous
 from core.domain import user_services
 from core.tests import test_utils
 import feconf
 import utils
 
 from google.appengine.api import urlfetch
 
+class ModifiedUserStatsAggregator(user_jobs_continuous.UserStatsAggregator):
```

Comment:
Isn't this the same as the one used in user_jobs_continuous_test? Couldn't you just import user_jobs_continuous_test directly instead of copying the code here?